server.js

```diff
@@ -1,34 +1,49 @@
-import express from 'express'
+import express, { request, response } from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import listEndpoints from 'express-list-endpoints'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+import booksData from './data/books.json'
+
+// Defines the port the app will run on. Defaults to 8080
 const port = process.env.PORT || 8080
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
+
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send(listEndpoints(app))
+})
+
+//Endpoint to get all books
+app.get('/books', (request, response) => {
+  response.json(booksData)
+})
+
+//Endpoint to get one book
+app.get('/books/:bookid', (request, response) => {
+  const { bookid } = request.params
+  const book = booksData.find(book => book.bookID === +bookid)
+  if (!book) {
+    response.status(404).send(`Sorry, book ${bookid} dosen't exist`)
+  }
+  response.json(book)
+})
+
+//Endpoint to get a specifik author/authors
+app.get('/author', (request, response) => {
+  const { authors } = request.query
+  if (authors) {
+    const authorsList = booksData.filter(author => author.authors.toLowerCase().includes(authors))
```

Comment:
You are filtering books by authors here, not the author list, so it should be the part of `/books` endpoint